packages/nextcloud/test/cookbook_test.dart

```diff
@@ -0,0 +1,245 @@
+import 'package:built_value/json_object.dart';
+import 'package:built_value_test/matcher.dart';
+import 'package:nextcloud/cookbook.dart' as cookbook;
+import 'package:nextcloud/nextcloud.dart';
+import 'package:nextcloud/src/api/cookbook.openapi.dart';
+import 'package:nextcloud_test/nextcloud_test.dart';
+import 'package:test/test.dart';
+import 'package:test_api/src/backend/invoker.dart' show Invoker;
+import 'package:universal_io/io.dart';
+
+void main() {
+  presets(
+    'cookbook',
+    'cookbook',
+    (preset) {
+      late DockerContainer container;
+      late NextcloudClient client;
+
+      setUpAll(() async {
+        container = await DockerContainer.create(preset);
+        client = await TestNextcloudClient.create(container);
+
+        final source = Directory('test/files/Recipes');
+        await client.webdav.mkcol(PathUri.parse(source.path.substring(11)));
+
+        for (final element in source.listSync(followLinks: false, recursive: true)) {
+          // remove the `test/files/` prefix
+          final path = PathUri.parse(element.path.substring(11));
+
+          if (element is Directory) {
+            await client.webdav.mkcol(path);
+          } else if (element is File) {
+            await client.webdav.putFile(
+              element,
+              element.statSync(),
+              path,
+            );
+          }
+        }
+
+        resetFixture();
+      });
+
+      tearDownAll(() async {
+        if (Invoker.current!.liveTest.errors.isNotEmpty) {
+          print(await container.allLogs());
+        }
+        container.destroy();
+      });
+
+      group(cookbook.$CategoriesClient, () {
+        // Get all known categories
+        //
+        //Future<BuiltList<Category>> listCategories() async
+        test('test listCategories', () async {
```

Comment:
The tests are really not like the other unit tests. Why does it have an extra "title", the signature of the method that is tested and then "test" as part of the description which is really unnecessary?

Reply:
Ah I see in the next commit this is changed. I think you accidentally added part of the tests to the wrong commit. In the setup step you also try to upload non-existent recipes for testing.